Lib/test/test_pty.py

```diff
@@ -292,7 +292,7 @@ def test_master_read(self):
         self.assertEqual(data, b"")
 
     def test_spawn_doesnt_hang(self):
-        pty.spawn([sys.executable, '-c', 'print("hi there")'])
+        pty.spawn([sys.executable, '-c', '1 + 1'])
```

Comment:
It may be a part of the manual testing. `pty.spawn()` runs subprocess and copy its stdout to this process stdout (fileno 1). The change that added this test ensured that `pty.spawn()` returns when reading from child returns empty bytes. But this test can implicitly test that `pty.spawn()` works at all and does not return before copying all child's output to stdout.

I try to automate this in #114647.